js/components/utilities/export.html

```diff
@@ -3,7 +3,20 @@
 <div data-bind="css: classes(), visible: !loading()">
     <div data-bind="css: classes('content')">
         <!-- ko if: canExport() -->
-            <textarea data-bind="css: classes('json-box'), textInput: $component.exportJSON"></textarea>
+            <div data-bind="css: classes('copy-clipboard')">
+                <button
+                    title="Copy to clipboard"
+                    data-bind="attr: {'id': 'btnCopyToClipboard' + $component.uniqueId, 'data-clipboard-target': '#exportJSON' + $component.uniqueId},
+                        css: classes({extra: 'btn btn-default btn-sm'}), click: copyExpressionToClipboard"
```

Comment:
Modify styles of button as it is in the cohort definition export tab (blue button with icon)